cmd/templates.go

```diff
@@ -14,34 +14,34 @@ mirror:
 {{- range $img := .AdditionalImages }}
     - name: {{ $img }}
 {{- end }}
-#
-# Example operators specification:
-#
-#  operators:
-#    - catalog: registry.redhat.io/redhat/redhat-operator-index:v{{ .Channel }}
-#      full: true
-#      packages:
-#        - name: ptp-operator
-#          channels:
-#            - name: 'stable'
-#        - name: sriov-network-operator
-#          channels:
-#            - name: 'stable'
-#        - name: cluster-logging
-#          channels:
-#            - name: 'stable'
```

Comment:
Why is the example block being removed? The intention was to provide example for users that are using the feature that allows for customization of the imageset file prior to running the oc-mirror command, such as adding their own desired operators

Reply:
added back